docs/guide/using-doctrine-orm-filters.php

```diff
@@ -43,7 +43,6 @@ public function getId(): ?int
      * Each Book is related to a User, supposedly allowed to authenticate.
      */
     #[ApiResource]
-    #[ApiFilter(UserFilter::class)]
```

Comment:
Why disabling it?

Reply:
This is not the goal of this guide, cf [this part in the doc](https://api-platform.com/docs/core/filters/#using-doctrine-orm-filters)